README.md

```diff
@@ -24,18 +17,80 @@ Change directory to the package under development. Note: an integration is a spe
 
 ```bash
 cd integrations
-cd package/my-integration
+cd package/my-package
 ```
 
-Run the `help` command and see available actions:
+Run the `help` command and see available commands:
 
 ```bash
 elastic-package help
 ```
 
-## GitHub authorization
+## Commands
+
+`elastic-package` current offers the commands listed below. 
+
+Some commands have a _global context_, meaning that they can be run from anywhere and they will have the 
+same result. Other commands have a _package context_; these must be run from anywhere under a package's
+root folder and they will operate on the contents of that package.
+
+For more details on a specific command, run `elastic-package help <command>`.
+
+### `elastic-package help`
+
+Use this command to get a listing of all commands available under `elastic-package` and a brief
+description of what each command does.
+
+_Context: global_
+
+### `elastic-package version`
+
+Use this command to print the version of `elastic-package` that you have installed. This is
+especially useful when reporting bugs.
+
+_Context: global_
+
+### `elastic-package build`
+
+Use this command to build a package. TODO: what is the purpose of building a package — when would
+a package developer want/need to run this command?
+
+_Context: package_
+
+### `elastic-package lint`
+
+Use this command to validate the contents of a package.
+
+_Context: package_
+
+### `elastic-package format`
+
+Use this command to format the contents of a package.
+
+_Context: package_
+
+### `elastic-package check`
+
+Use this command to run the `format`, `lint`, and `build` all at once, in that order.
+
+_Context: package_
+
+### `elastic-package stack`
+
+Use this command to spin up a Docker-based Elastic Stack consisting of Elasticsearch, Kibana, and 
+the Package Registry.
+
+_Context: global_
+
+### `elastic-package promote`
+
+Use this command to promote a package to the Snapshot Package Registry.
```

Comment:
@mtojek Is this description accurate?

Reply:
The promotion from Integrations to package-storage happens automagically (Jenkins job).

The `promote` command exposes a package selection form in the TUI, which simplifies promotion between different stages (snapshot, staging, production) including opening relevant pull requests to the package-storage.

Reply:
Ah yes, thanks for reminding me! I can fix the description. Is this command one we want to document or is it meant more for administrators only? We can also document it but say that it's not meant for general use or something like that.

Reply:
Hm, I wouldn't keep it secret (do not mention in README), but it's valid point to add a notice about administrator use only.

Reply:
Sounds good, thanks. I will add a notice about administrator use only.

Reply:
Added in 291401e.